metadeploy/api/migrations/0113_auto_20210729_1954.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.1.12 on 2021-07-29 19:54
+
+import django.core.serializers.json
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0112_merge_20201130_1757"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="job",
+            name="results",
+            field=models.JSONField(blank=True, default=dict),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="preflight_checks",
+            field=models.JSONField(blank=True, default=list),
+        ),
+        migrations.AlterField(
+            model_name="preflightresult",
+            name="results",
+            field=models.JSONField(blank=True, default=dict),
+        ),
+        migrations.AlterField(
+            model_name="scratchorg",
+            name="config",
+            field=models.JSONField(
+                blank=True,
+                encoder=django.core.serializers.json.DjangoJSONEncoder,
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="step",
+            name="source",
+            field=models.JSONField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name="step",
+            name="task_config",
+            field=models.JSONField(blank=True, default=dict),
+        ),
+        migrations.AlterField(
+            model_name="user",
+            name="first_name",
+            field=models.CharField(
+                blank=True, max_length=150, verbose_name="first name"
+            ),
+        ),
```

Comment:
Weird, not sure why Django thinks this field has changed 🤔

Reply:
I saw a migration run locally on the `auth` module, so I think this was triggered by a py dep upgrade (I assume django-allauth?).

Reply:
```
auth
 [X] 0001_initial
 [X] 0002_alter_permission_name_max_length
 [X] 0003_alter_user_email_max_length
 [X] 0004_alter_user_username_opts
 [X] 0005_alter_user_last_login_null
 [X] 0006_require_contenttypes_0002
 [X] 0007_alter_validators_add_error_messages
 [X] 0008_alter_user_username_max_length
 [X] 0009_alter_user_last_name_max_length
 [X] 0010_alter_group_name_max_length
 [X] 0011_update_proxy_permissions
 [X] 0012_alter_user_first_name_max_length
```

Reply:
Ah, makes sense. Looks like Django did it